Makefile

```diff
@@ -630,47 +630,47 @@ package-capi:
 	fi
 	
 	if [ -f target/headless/$(CARGO_TARGET)/release/wasmer.dll ]; then \
-		cp target/headless/$(CARGO_TARGET)/release/wasmer.dll package/lib/wasmer-headless.dll ;\
+		cp target/headless/$(CARGO_TARGET)/release/wasmer.dll package/lib/wasmer-headless2.dll ;\
```

Comment:
What is this headless 2? @fschutt

Reply:
It was supposed to prevent a naming conflict because we already ship `libwasmer-headless.a` for `linux-aarch64`, although I don't know if the feature flags are the same as for all other -headless libraries. It's supposed to prevent a naming conflict.

I've marked this PR as a draft for now until I've properly figured out why the packaging doesn't work, I'll ping you once I'm done.

Reply:
Okay, it's packaged correctly now.